v1/errors/config_test.go

```diff
@@ -0,0 +1,17 @@
+package errors
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
Shouldn't we use a testify test suite here in order to comply with what was decided a couple of month ago ?

Reply:
IMO a test suite makes sense if you use a pivot object on which you implement methods or if you need state variables shared across all tests, but otherwise it's a bit overkill. The idea with testify was to settle on a common framework, really...

Reply:
Copy that